build.gradle.kts

```diff
@@ -169,6 +169,12 @@ configurations.all {
     }
 }
 
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(21))
+    }
+}
```

Comment:
The one that we should be modifying is the one like in the `RewriteJavaPlugin` class instead.